source/agent/quic/index.js

```diff
@@ -252,14 +242,14 @@ module.exports = function (rpcClient, selfRpcId, parentRpcId, clusterWorkerIP) {
             return callback('callback', {type: 'failed', reason: 'Create Connection failed'});
         }
 
-        connections.addConnection(connectionId, connectionType, options.controller, conn, 'out')
+        router.addLocalDestination(connectionId, connectionType, conn)
         .then(onSuccess(callback), onError(callback));
     };
 
     that.unsubscribe = function (connectionId, callback) {
         log.debug('unsubscribe, connectionId:', connectionId);
-        var conn = connections.getConnection(connectionId);
-        connections.removeConnection(connectionId).then(function(ok) {
+        var conn = router.getConnection(connectionId);
+        router.removeConnection(connectionId).then(function(ok) {
             if (conn && conn.type === 'internal') {
```

Comment:
Same as the 'removeConnection' call in `unpublish`

Reply:
Fixed.